images/linux/scripts/installers/containers.sh

```diff
@@ -4,6 +4,8 @@
 ##  Desc:  Installs container tools: podman, buildah and skopeo onto the image
 ################################################################################
 
+source $HELPER_SCRIPTS/os.sh
```

Comment:
Looks like we even don't need any new repository for 20.04. See installation guide: https://podman.io/getting-started/installation

Let's get rid of it

Reply:
Sadly, but `Ubuntu20 `still requires source repo to install these tools